src/scene/scenegraphnode.cpp

```diff
@@ -246,6 +258,11 @@ std::unique_ptr<SceneGraphNode> SceneGraphNode::createFromDictionary(
         LDEBUG(fmt::format(
             "Successfully created renderable for '{}'", result->identifier()
         ));
+
+        // If the renderable child has a bounding sphere that is larger, we allow it to override
+        result->_boundingSphere.setValue(std::max(result->_renderable->boundingSphere(),
+            result->_boundingSphere.value()
```

Comment:
I do think we should present a warning if we override a value specified by the user in the dictionary

Reply:
Added warning :)